app/views/cart/_orderables.html.slim

```diff
@@ -1,51 +1,52 @@
-= form_with(url: cart_remove_path) do |f|
- - @cart.orderables.each do |orderable|
-  - product = orderable.product
-  |  <div id="
-  = "cart-#{product.id}"
-  | "> 
-  div[style=" display:flex; justify-content:space-between: margin: 0em 5em;"]
-    strong
-      | Name: 
-    = product.name 
-  br
-    / strong
-    /   | Price
-    / = product.price
-    .form-group.mb-3
-    = f.label :customer_name
-    = f.text_field :customer_name, value: current_user.profile&.first_name, class: 'form-control form-control-sm'
-    .form-group.mb-3
-    = f.label :address
-    = f.text_area :address,value: current_user.profile&.address, class: 'form-control form-control-sm'
-    .form-group.mb-3
-    = f.label :qunatity
-    = f.number_field :quantity, value: orderable.quantity ,readonly: true,class: 'form-control form-control-sm'
-    / = f.submit 
-    = product.name
-    |  * 
-    = orderable.quantity
-    
-    |  = 
-    = product.price
-    |  * 
-    = orderable.quantity
-    |  = 
-    = orderable.total
-    |  rs 
+div2[style="display:flex;  max-width: 600px; margin: auto; justify-content:space-between: margin: 0em 5em;"]
+ = form_with(url: cart_remove_path) do |f|
+  - @cart.orderables.each do |orderable|
+    - product = orderable.product
+    |  <div id="
+    = "cart-#{product.id}"
+    | "> 
     br
-    / -if orderable.quantity > 2 && @cart.apply_discount > orderable.qty_discount
-    /   | After applying discount 5%
-    /   h3
-    /     u
-    /     = number_with_precision((orderable.qty_discount), precision: 2)
-    /     |  rs 
-   
-    
-    = form_with(url: cart_remove_path) do |f|
-      = f.hidden_field :id, value: orderable.id
-      = f.submit "remove" 
-      = link_to "refresh" , cart_path
-     
+      strong
+        | Name: 
+      = product.name 
     br
+      / strong
+      /   | Price
+      / = product.price
+      .form-group.mb-3
+      = f.label :customer_name
+      = f.text_field :customer_name, value: current_user.profile&.first_name, class: 'form-control form-control-sm'
+      .form-group.mb-3
+      = f.label :address
+      = f.text_area :address,value: current_user.profile&.address, class: 'form-control form-control-sm'
+      .form-group.mb-3
+      = f.label :qunatity
+      = f.number_field :quantity, value: orderable.quantity ,readonly: true,class: 'form-control form-control-sm'
+      / = f.submit 
+      = product.name
+      |  * 
+      = orderable.quantity
+      
+      |  = 
+      = product.price
+      |  * 
+      = orderable.quantity
+      |  = 
+      = orderable.total
+      |  rs 
+      br
+      / -if orderable.quantity > 2 && @cart.apply_discount > orderable.qty_discount
```

Comment:
commented section contain fixed conditions and no more required. pls remove from code

Reply:
ok sir